src/library_async.js

```diff
@@ -41,15 +41,14 @@ addToLibrary({
 #if ASYNCIFY_DEBUG
       dbg('asyncify instrumenting imports');
 #endif
-      var importPatterns = [{{{ ASYNCIFY_IMPORTS.map(x => '/^' + x.split('.')[1].replace(new RegExp('\\*', 'g'), '.*') + '$/') }}}];
+      var importPattern = {{{ new RegExp(`^(${ASYNCIFY_IMPORTS.map(x => x.split('.')[1]).join('|').replace(/\*/g, '.*')})$`) }}};
```

Comment:
Should the `new RegExp` part not happen at runtime?   The {{{ }}} macro expander will stringify the result.. what does that even look like for a regex object?

Reply:
Looks like you end up with just the `/ .. /`  literal.  Is that want you want?  If so, maybe just drop the new RegExp thing completely?

```
$ node
> a = new RegExp('^sam')
> '' + a
> '/^sam/'
```

Reply:
> Looks like you end up with just the / .. / literal. Is that want you want?

Yes, and that's what the code did before for individual items. I kept `new RegExp` for consistency with what we used to have, but also because it provides at least basic compile-time validation for the regexp (it would fail during compilation if there were some invalid symbols).